### PR TITLE
Create PancakeSort.cpp

### PR DESCRIPTION
**Pull request by kashika3101**
Implemented a sorting technique colloquial for mathematical problem for sorting a disorder of pancakes in order of size.

### Have you read the [Contributing Guidelines on Pull Requests](https://github.com/TesseractCoding/NeoAlgo/blob/master/CONTRIBUTING.md#pull-requests)?

Yes

### Description

An important sorting technique using flip operation analogous to flipping pancakes.

### Checklist

- [x] I've read the contribution guidelines.
- [x] I've checked the issue list before deciding what to submit.
- [ ] I've edited the `README.md` and link to my code.

## Related Issues or Pull Requests

#254

### CODE REVIEW

**Comment by kashika3101** [comment]
please review @atarax665

File: C-Plus-Plus/ds/PancakeSort.cpp [hunk 1]
```diff
@@ -0,0 +1,72 @@
+#include <iostream>
```
Comment (atarax665) [inline C-Plus-Plus/ds/PancakeSort.cpp:1]:
```suggestion

```
No need of adding iostream as its already included in bits/stdc++ .

File: C-Plus-Plus/ds/PancakeSort.cpp [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+    int number;
+    cin>>number;
+
+    int arr[number];                          //creating an array
+    for (int i = 0; i < number; ++i)         //input the array elements
+        cin>>arr[i];
+      
+    PancakeSort(arr,number);
+    return 0;
+}
```
Comment (atarax665) [inline C-Plus-Plus/ds/PancakeSort.cpp:72]: Add sample input/output, space and time complexity in commented form.

File: C-Plus-Plus/ds/PancakeSort.cpp [hunk 3]
```diff
@@ -0,0 +1,72 @@
+#include <iostream>
+#include<bits/stdc++.h>
```
Comment (atarax665) [inline C-Plus-Plus/ds/PancakeSort.cpp:2]: Add description of Pancake Sort in commented form at the top.

**Review by atarax665 — CHANGES_REQUESTED** [review]

**Comment by atarax665** [comment]
Make appropriate changes to README.md file of C++ folder.

**Pull request closed without merging**